docs/spec.md

```diff
@@ -2421,7 +2424,7 @@ the XLA compiler. In the future, we are planning to unify this metadata
 | (I1)  | `inputs`              | variadic number of values                     |
 | (I2)  | `call_target_name`    | constant of type `string`                     |
 | (I3)  | `has_side_effect`     | constant of type `i1`                         |
-| (I4)  | `backend_config`      | constant of type `string`                     |
+| (I4)  | `backend_config`      | constant of type `string` dictionary with key of type `string` and values of type `Value`                   |
```

Comment:
```suggestion
| (I4)  | `backend_config`      | dictionary with key of type `string` and values of type `Value`                   |
```

Reply:
Agreed, didn't realize we were supporting both.

Reply:
changed it to "attribute dictionary"